communication/contributor-comms/role-handbooks/Comms-Events.md

```diff
@@ -14,8 +14,8 @@ Time Commitment:
 ## Minimum skills and Requirements
 
 - Kubernetes github org [membership](https://github.com/kubernetes/community/blob/master/community-membership.md)
-- familiarity with the workflow of [#sig-contributor-comms](https://github.com/kubernetes/community/tree/master/communication/marketing-team)
-- has understanding of the [social guidlines](https://github.com/kubernetes/community/blob/master/communication/marketing-team/storytelling-resources/social-guidelines.md)
+- familiarity with the workflow of [#sig-contributor-comms](https://github.com/kubernetes/community/tree/master/communication/contributor-comms)
```

Comment:
Is it possible to go through this document and fix up all the links that link into this same repo as per below?
```suggestion
- familiarity with the workflow of [#sig-contributor-comms](/communication/contributor-comms)
```

Reply:
I'd do the same. Thanks!